trufflescripts/revert.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable */
+const { getCurrentBlock, revertSnapshot } = require('./utils')(web3)
+
+const argv = require('minimist')(process.argv.slice(2), { string: 'a' })
+
+/**
+ * truffle exec trufflescripts/claim_funds.js
+ * to claim funds for the current auction for both seller and buyer,
+ * from auction's sellerBalances and buyerBalances respectively
+ * @flags:
+ * --seller                     sellerBalance for seller only
+ * --buyer                      buyerBalance for buyer only
+ * -a seller|buyer|<address>    for the given address
+ * -i <index>                   for auction with given index
+ * --last                       for last auction
+ */
+
+module.exports = async () => {
+  let blockID
+
+  blockID = argv.b
+  if(argv.b === undefined) blockID = '0x01'
```

Comment:
Can be written as `const blockID = argv.b || '0x01'`
or `const blockID = argv.b !== undefined ? argv.b || '0x01'` if you want to check for undefined